Protocol.md

```diff
@@ -104,12 +104,18 @@ connection.
 
 ## Server's Permanent Key
 
-A SaltyRTC complicant server SHOULD have a permanent NaCl key pair for
-public key authenticated encryption. If the server has such a key pair,
-it will be used to sign<sup>1</sup> the server's session key and the
-client's permanent key to mitigate man-in-the-middle attacks. In order
-to validate this signature, a client that connects to a server SHOULD
-know the server's public permanent key.
+A SaltyRTC complicant server SHOULD have at least one permanent NaCl key
+pair for public key authenticated encryption. If the server has such a
```

Comment:
I wonder if we should remove the "for public key authenticated encryption" part.